parasoft-findings-teamcity-agent/src/main/java/com/parasoft/findings/teamcity/agent/ParasoftFindingsBuildProcess.java

```diff
@@ -227,12 +234,26 @@ private void transform(File from, File to, String xslFile, File checkoutDir) {
 
             if (checkHasContent(to) && !_transformFailed) {
                 // Send a notification to TC that a JUnit report is ready to be consumed.
-                // This allows running the plug-in build step without having to configure 
+                // This allows running the plug-in build step without having to configure
                 // the XML Report Processing build feature in a TC project.
                 _build.getBuildLogger().message("Wrote transformed report to " + to.getAbsolutePath());
                 String relativePath = checkoutDir.toURI().relativize(to.toURI()).getPath();
-                _build.getBuildLogger().logMessage(DefaultMessagesInfo.createTextMessage
-                        ("##teamcity[importData type='junit' path='"+relativePath + "']"));
+                String type = "";
+                if(to.getName().startsWith(PMD_PREFIX)) {
```

Comment:
Use "switch" for condition and use "isEmpty()" to check empty string. 
When the format doesn't match I think we can use existing `reportUnexpectedFormat `method